sql/src/main/java/io/crate/metadata/information/InformationKeyColumnUsageTableInfo.java

```diff
@@ -59,27 +63,28 @@
         static final Reference TABLE_SCHEMA = createRef(Columns.TABLE_SCHEMA, DataTypes.STRING);
         static final Reference TABLE_NAME = createRef(Columns.TABLE_NAME, DataTypes.STRING);
         static final Reference COLUMN_NAME = createRef(Columns.COLUMN_NAME, DataTypes.STRING);
-        static final Reference ORDINAL_POSITION = createRef(Columns.ORDINAL_POSITION, DataTypes.SHORT);
+        static final Reference ORDINAL_POSITION = createRef(Columns.ORDINAL_POSITION, DataTypes.INTEGER);
```

Comment:
why the change here? are the column-types of the information schema part of the SQL standard?

Reply:
I was looking at https://docs.microsoft.com/en-us/sql/relational-databases/system-information-schema-views/key-column-usage-transact-sql which uses INTEGER.

Reply:
The ordinal is a `cardinal_number`: https://www.postgresql.org/docs/8.4/static/infoschema-key-column-usage.html

http://www.npgsql.org/dev/types.html at the bottom:

>        cardinal_number: int4 (must be nonnegative or null)